src/apps/relay/mainrelay.h

```diff
@@ -192,6 +192,7 @@ typedef struct _turn_params_ {
   char ca_cert_file[1025];
   char cert_file[1025];
   char pkey_file[1025];
+  char rpk_enabled;
```

Comment:
please use `bool` from the `stdbool` header.

i know other "bool"-like variables don't currently use bool, but that's being worked on in other PRs.

Reply:
fixed